### PR TITLE
Subtitle color changed in lists

### PR DESCRIPTION
**Pull request by yceballost**
## 🎟️ **Jira ticket**
https://jira.tid.es/browse/IOS-9929

### CODE REVIEW

**Comment by amegias** [comment]
Let us check this new feature:
https://github.com/Telefonica/mistica-ios/blob/main/CONTRIBUTING.md#record-new-snapshot-testing-reference-screenshots-for-a-pr
https://github.com/Telefonica/mistica-ios/pull/342

**Comment by amegias** [comment]
> Let us check this new feature: https://github.com/Telefonica/mistica-ios/blob/main/CONTRIBUTING.md#record-new-snapshot-testing-reference-screenshots-for-a-pr #342

@franciscojrp  we need to review this.
Yayo changed the subtitles and I've executed /record-screenshots.
The screenshots were re-recorded (196d56bb6402fc251ebd37381d7ad695f584e819) but:
- one test still failed => ???
- the PR should be updated showing ONLY one test failed, shouldn't it?

File: Sources/Mistica/Components/Lists/Internals/CellCenterSectionView.swift [hunk 1]
```diff
@@ -30,10 +30,15 @@ class CellCenterSectionView: UIStackView {
         }
     }
 
-    var subtitleTextColor: UIColor = .textSecondary {
+    var subtitleTextColor: UIColor = .textPrimary {
         didSet {
             subtitleLabel.textColor = subtitleTextColor
-            detailLabel.textColor = subtitleTextColor
```
Comment (amegias) [inline Sources/Mistica/Components/Lists/Internals/CellCenterSectionView.swift:36]: we were using the same color for both labels => wrong

**Comment by franciscojrp** [comment]
> > Let us check this new feature: https://github.com/Telefonica/mistica-ios/blob/main/CONTRIBUTING.md#record-new-snapshot-testing-reference-screenshots-for-a-pr #342
> 
> @franciscojrp we need to review this. Yayo changed the subtitles and I've executed /record-screenshots. The screenshots were re-recorded ([196d56b](https://github.com/Telefonica/mistica-ios/commit/196d56bb6402fc251ebd37381d7ad695f584e819)) but:
> 
> * one test still failed => ???

The testTabBarControl test seems to be the problem, we have previously seen it fail in another PR, and we did not see the cause, but if it keeps failing for every PR we need to review it 

> * the PR should be updated showing ONLY one test failed, shouldn't it?

Good point, we need to change that if possible

> * the tests have been recorded for every comment :S :S (https://github.com/Telefonica/mistica-ios/actions/runs/8440138081/job/23116358692)

This is pretty weird, let's see what happens with this comment

**Comment by amegias** [comment]
@yceballost Could we continue here https://github.com/Telefonica/mistica-ios/pull/371 please?

**Comment by yceballost** [comment]
> @yceballost Could we continue here #371 please?

sure! Close this PR! Thanks 🫶

**Pull request closed without merging**